libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -628,6 +644,15 @@ AP_GPS_UBLOX::_parse_gps(void)
                     break;
                 }
             }
+        } else if(_msg_id == MSG_ACK_NACK) {
+            switch(_buffer.ack.clsID) {
+            case CLASS_CFG:
+                switch(_buffer.ack.msgID) {
+                case MSG_CFG_SBAS:
+                    _unconfigured_messages &= ~CONFIG_SBAS;
```

Comment:
Yes, actually we have that.

Actually what we should do is that should be entirely optional depending on the GPS_SBAS option. We shouldn't be messing with it if the GPS_SBAS is 2. We should just mark the check as passed and move on. This is in someways an existing bug I'd say, as we shouldn't ask for the SBAS if we aren't going to do anything with it.